scripts/libraries/mnemonic-signer.ts

```diff
@@ -0,0 +1,28 @@
+import { ethereum, secp256k1, mnemonic, hdWallet } from '@zoltu/ethereum-crypto'
+import { Bytes } from '@zoltu/ethereum-types';
+
+export class MnemonicSigner {
+	private constructor(
+		public readonly words: string[],
+		public readonly privateKey: bigint,
+		public readonly publicKey: secp256k1.AffinePoint & secp256k1.JacobianPoint,
+		public readonly address: bigint,
```

Comment:
Seems worthwhile to hex this up, otherwise i'm doing it by hand whenever i'm debugging, but it's not strictly necessary

Reply:
Going to leave this unchanged for this PR, but will consider adding a way to extract the hex string version of this in a future PR.